Makefile

```
@@ -33,6 +32,10 @@ ORG=openshift
 TAG=$(shell git rev-parse --abbrev-ref HEAD)
 IMAGE=$(REGISTRY)/$(ORG)/origin-cluster-node-tuning-operator:$(TAG)
 
+# PAO variables
+CLUSTER ?= "ci"
+PAO_CRD_APIS :=$(addprefix ./pkg/apis/pao/,v2 v1 v1alpha1)
```

Maybe we should use `API_TYPES_DIR` here?
```suggestion
PAO_CRD_APIS :=$(addprefix ./$(API_TYPES_DIR)/pao/,v2 v1 v1alpha1)
```